engine/apps/slack/models/slack_message.py

```diff
@@ -72,26 +102,21 @@ class Meta:
 
     @property
     def slack_team_identity(self):
-        if self._slack_team_identity is None:
-            if self.organization is None:  # strange case when organization is None
-                logger.warning(
-                    f"SlackMessage (pk: {self.pk}) fields _slack_team_identity and organization is None. "
-                    f"It is strange!"
-                )
```

Comment:
I wasn't able to find any mentions of this log message for the last 90 days + there's no git history available unfortunately for why this is here. As I am proposing to (eventually) drop `slack_team_identity` from this model, I propose to remove this